Documentation/Plus Codes Specification.md

```diff
@@ -0,0 +1,119 @@
+
+A significant digit in places 11 and higher represents:
+
+* an offset north equal to the north-south distance for that code length multiplied by (the significant digit value divided by four and rounded down)
+* an offset east equal to the west-east distance for that code length multiplied by (the significant digit value modulo 4)
+
+<kbd>N</kbd><kbd>E</kbd><kbd>N</kbd><kbd>E</kbd><kbd>N</kbd><kbd>E</kbd><kbd>N</kbd><kbd>E</kbd><kbd>+</kbd><kbd>N</kbd><kbd>E</kbd><kbd>X</kbd><kbd>X</kbd><kbd>X</kbd><kbd>X</kbd><kbd>X</kbd>
+
+:information_source: Note that latitudes greater than 90° N do not exist and therefore the digit place 1 MUST only have values 0–8. Likewise, digit place 2 MUST only have values 0–17.
+
+## Full code
+
+A **full code** Plus Code is globally usable and requires no other reference to interpret.
+
+Every full code MUST include significant digits in digit places 1 up through the code length. If the code length is less than 8, then the padding character is placed in digit places after the last significant digit up to and including the 8th digit place. The format separator is added after the 8th digit place.
+
+Therefore, the set of full codes exactly matches the [Perl Compatible Regular Expression](http://pcre.org):
+
+````perl
+^[2-9C][2-9CFGHJMPQRV](0{6}\+|[2-9CFGHJMPQRVWX]{2}(0000\+|[2-9CFGHJMPQRVWX]{2}(00\+|[2-9CFGHJMPQRVWX]{2}\+([2-9CFGHJMPQRVWX]{2,7})?)))$
+````
+
+## Short code
+
+A **short code** Plus Code is meaningful only when the producer and consumer of the Plus Code can agree on the approximate latitude and longitude of a **reference location**. A short code can be easier to use and remember than a full code.
+
+Plus Codes with code length less than 8 SHALL NOT be represented as short codes.
+
+The latitudinal distance (in degrees) and longitudinal distance (in degrees) between the Plus Code area's center and the reference location determines which digit places can be omitted in the short code representation:
+
+* If both distances are less than 10 degrees, digit places 1–2 MAY be omitted.
+* If both distances are less than 0.5 degrees, digit places 1–2 or 1–4 MAY be omitted.
+* If both distances are less than 1/40 degrees, digit places 1–2, 1–4 or 1–6 MAY be omitted.
+
+A short code represents the unique full code Plus Code nearest (latitudinal distance + longitudinal distance) to the reference location.
+
+:information_source: Note that omitted digits places of a short code will not necessarily be the same as the reference location.
+
+Therefore, the set of short code representations exactly matches the Perl Compatible Regular Expression:
+
+```perl
+/^([2-9CFGHJMPQRVWX]{2})?([2-9CFGHJMPQRVWX]{2})?[2-9CFGHJMPQRVWX]{2}\+([2-9CFGHJMPQRVWX]{2,7})?$/
+```
+
+## Copyright
+
+The authors dedicate this document to the public domain under [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
I'm not sure this is the proper way to copyright this, as it includes content that has previously been made available under Apache-2.0 license.

Reply:
Thank you! Deleted.

:white_check_mark: Fixed at d7c6d2507ce49c2a0ef0d7c1d5249ef4b668db37